.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,366 @@
+
+      - name: 📤 Sync files to OSS repository
+        run: |
+          cd /tmp/maria-repo
+          
+          echo "🧹 Cleaning existing files (except .git and .github)..."
+          find . -maxdepth 1 -not -name '.git' -not -name '.github' -not -name '.' -exec rm -rf {} \;
+          
+          echo "📋 Copying new files from maria_code..."
+          cp -r /tmp/maria-oss/* . 2>/dev/null || true
+          cp -r /tmp/maria-oss/.[^.]* . 2>/dev/null || true
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)